src/Sentry/PlatformAbstractions/NetFxInstallationsEventProcessor.cs

```diff
@@ -0,0 +1,68 @@
+#if NETFX
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using Sentry.Extensibility;
+using Sentry.PlatformAbstractions;
+
+namespace Sentry.PlatformAbstractions
+{
+    internal class NetFxInstallationsEventProcessor : ISentryEventProcessor
+    {
+        internal static readonly string NetFxInstallationsKey = ".NET Framework";
+
+        private readonly Lazy<Dictionary<string, string>> _netFxInstallations =
+            new Lazy<Dictionary<string, string>>(() => GetInstallationsDictionary(), LazyThreadSafetyMode.ExecutionAndPublication);
+
+        private bool _netFxInstallationEnabled = true;
+
+        private readonly SentryOptions _options;
+
+        internal NetFxInstallationsEventProcessor(SentryOptions options)
+        {
+            Debug.Assert(options != null);
+            _options = options;
+        }
```

Comment:
nullable reference types now are in

```suggestion
        internal NetFxInstallationsEventProcessor(SentryOptions options) => _options = options;
```